tests/outofrange_queries.test/queries

```diff
@@ -45,6 +45,7 @@ alter table t {schema {
     cstring c[4] null=yes
     u_short us null=yes
     u_int ui null=yes
```

Comment:
Double check the intent here.  The keys referenced this column without it being present.